js/parts-more/RadialAxis.js

```diff
@@ -8,13 +8,29 @@
  *
  * */
 'use strict';
+var __read = (this && this.__read) || function (o, n) {
```

Comment:
This is a polyfill for array destructuring assignments. At the time being we need to avoid those descructuring assignments in order to prevent our modules to be bloated with this polyfill.

Reply:
Sorry, I previously missed that destructuring assignment. Corrected.